src/tox/venv.py

```diff
@@ -694,6 +694,28 @@ def tox_testenv_create(venv, action):
 
 def cleanup_for_venv(venv):
     within_parallel = PARALLEL_ENV_VAR_KEY in os.environ
+    # if the directory exists and it doesn't look like a virtualenv, produce
+    # an error
+    if venv.path.exists():
+        dir_items = set(os.listdir(str(venv.path))) - {".lock", "log"}
+        dir_items = {p for p in dir_items if not p.startswith(".tox-")}
+    else:
+        dir_items = set()
+
+    if not (
+        # doesn't exist => OK
+        not venv.path.exists()
+        # does exist, but it's empty => OK
+        or not dir_items
+        # it exists and we're on windows with Lib and Scripts => OK
+        or (INFO.IS_WIN and dir_items > {"Scripts", "Lib"})
+        # non-windows, with lib and bin => OK
+        or dir_items > {"bin", "lib"}
+    ):
+        venv.status = "error"
+        reporter.error("cowardly refusing to delete `envdir`: {}".format(venv.path))
```

Comment:
How about explaining why we are cowardly refusing? E.g. add 

"- directory contents do not look like a virtual environment."

Reply:
updated!

```console
$ tox --devenv docs
py create: /tmp/tox/docs
ERROR: cowardly refusing to delete `envdir` (it does not look like a virtualenv): /tmp/tox/docs
___________________________________ summary ____________________________________
ERROR:   py: error
```